browser/src/layer/marker/A11yTextInput.js

```diff
@@ -226,29 +226,32 @@ L.A11yTextInput = L.TextInput.extend({
 			eventDescription += _('Out of table') + '. ';
 		}
 		for (i = 0; i < inList.length; i++) {
-			eventDescription += _('Table with') + ' ' + inList[i].rowCount + ' ' + _('rows') + ' '
-				+ _('and') + ' ' + inList[i].colCount + ' ' + _('columns') + '. ';
+			eventDescription += _('Table with {0} rows and {1} columns').replace('{0}', inList[i].rowCount).replace('{1}', inList[i].colCount) + '. ';
 		}
 		if (this._lastRowIndex !== row || this._lastRowSpan !== rowSpan) {
 			this._lastRowIndex = row;
-			eventDescription += _('Row') + ' ' + row;
 			if (this._lastRowSpan !== rowSpan) {
 				if (rowSpan > 1) {
-					eventDescription += ' ' + _('through') + ' ' + (row + rowSpan - 1) ;
+					eventDescription += _('Row {0} through {1}').replace('{0}', row).replace('{1}', row + rowSpan - 1);
 				}
 				this._lastRowSpan = rowSpan;
```

Comment:
When the code ends up here in case of:
- `this._lastRowSpan !== rowSpan` is true
- `rowSpan > 1` is false

...in the previous version of the code, `_('Row') + ' ' + row` is already appended to `eventDescription`, but not in the changed code.